run/types/DeviceWrapper.ts

```diff
@@ -1051,28 +1071,32 @@ export class DeviceWrapper {
     return this.toShared().getAttribute(attribute, elementId);
   }
 
-  public async disappearRadioButtonSelectediOS(timeOption: DMTimeOption) {
-    const radioButton = await this.waitForTextElementToBePresent({
-      strategy: 'accessibility id',
-      selector: `${timeOption} - Radio`,
-    });
-    const attr = await this.getAttribute('value', radioButton.ELEMENT);
-    if (attr === 'selected') {
-      console.log('Great success - default time is correct');
-    } else {
-      throw new Error('Dammit - default time was not correct');
-    }
-  }
-  public async disappearRadioButtonSelectedAndroid(timeOption: DMTimeOption) {
-    const radioButton = await this.waitForTextElementToBePresent({
-      strategy: 'accessibility id',
-      selector: timeOption,
-    });
-    const attr = await this.getAttribute('selected', radioButton.ELEMENT);
-    if (attr) {
-      console.log('Great success - default time is correct');
+  public async disappearRadioButtonSelected(
+    platform: SupportedPlatformsType,
+    timeOption: DISAPPEARING_TIMES
+  ) {
+    if (platform === 'ios') {
+      const radioButton = await this.waitForTextElementToBePresent({
+        strategy: 'accessibility id',
+        selector: `${timeOption} - Radio`,
+      });
+      const attr = await this.getAttribute('value', radioButton.ELEMENT);
+      if (attr === 'selected') {
+        console.log('Great success - default time is correct');
+      } else {
+        throw new Error('Dammit - default time was not correct');
+      }
     } else {
-      throw new Error('Dammit - default time was not correct');
+      const radioButton = await this.waitForTextElementToBePresent({
+        strategy: 'accessibility id',
+        selector: timeOption,
+      });
+      const attr = await this.getAttribute('selected', radioButton.ELEMENT);
+      if (attr) {
+        console.log('Great success - default time is correct');
+      } else {
+        throw new Error('Dammit - default time was not correct');
+      }
```

Comment:
just a personal preference, but usually you want to do the precondition checks and then the normal flow
so instead of 
```ts
this.getAttributes
if(attr) {
	 log
 } else {
	throw
}
etc
```

it's easier to read
```ts
if(!attr) {
	throw	
}
log
etc
...

this.getAttributes

Reply:
If I do `this.getAttributes` after the precondition check I won't have the `attr` ?
I understand the `(!attri)` first but not the actual check...

Reply:
oops, yeah. What I meant was what how you've changed it in the end so all good.